### PR TITLE
Seek 1.11 licenses

### PR DESCRIPTION
**Pull request by stuzart**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -76,5 +76,8 @@ class Application < Rails::Application
 
     # Ignore translation overrides when testing
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', 'overrides', '**', '*.{rb,yml}')] unless Rails.env.test?
+
+    config.recommended_data_licenses = []
```
Comment (stuzart) [inline config/application.rb:80]: this I think will overwrite the config whenever the app is restarted. The defaults can instead go in the default settings @ /config/initializers/seek_configuration.rb - and should the default be the fall-back setting ?

File: app/views/workflows/edit.html.erb [hunk 2]
```diff
@@ -26,7 +26,7 @@
       <p class="help-block">This field is used to indicate to users what level of stability they can expect from the workflow.</p>
     </div>
 
-    <%= render partial: 'assets/license_selector', locals: { resource: @workflow, license_group: :software, default_license: 'Apache-2.0' } %>
+    <%= render partial: 'assets/license_selector', locals: { resource: @workflow, license_group: :software, recommended_licenses: Seek::Config.recommended_software_licenses, default_license: 'Apache-2.0' } %>
```
Comment (stuzart) [inline app/views/workflows/edit.html.erb:29]: the recommended licenses can be determined inside the partial, based on the license_group. currently this only affects workflows

File: lib/seek/config.rb [hunk 3]
```diff
@@ -15,6 +15,14 @@ def dm_project_name_fallback
       project_name
     end
 
+    def recommended_data_licenses_fallback
+      ['CC-BY-4.0']
+    end
+    
+    def recommended_software_licenses_fallback
+      ['CC0-1.0']
```
Comment (stuzart) [inline lib/seek/config.rb:23]: workflow uses Apache-2 as the default license

File: app/views/workflows/edit.html.erb (hunk 2 above)
Comment (alaninmcr) [reply app/views/workflows/edit.html.erb:29]: I agree with Stuart

**Comment by stuzart** [comment]
merged

**Pull request closed without merging**